server/api/account.ts

```diff
@@ -0,0 +1,61 @@
+import express from 'express';
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+require('dotenv').config()
```

Comment:
This needs to be moved much higher in your app otherwise dotenv will only be configured in this file? Move to your express app index file

Reply:
Other things commonly stored in .env include DB url, JSON secrets (which you will need anywhere that you need to verify them), and hostname / port